message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1708,5 +1708,21 @@
       <field type="float" name="min_distance" units="m">Minimum distance the sensor can measure.</field>
       <field type="float" name="max_distance" units="m">Maximum distance the sensor can measure.</field>
     </message>
+    <message id="11038" name="WATER_DEPTH">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change and should NOT be used in stable production environments -->
+      <description>Water depth</description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot)</field>
+      <field type="uint8_t" name="id" instance="true">Onboard ID of the sensor</field>
+      <field type="uint8_t" name="healthy">Sensor data healthy (0=unhealthy, 1=healthy)</field>
+      <field type="int32_t" name="lat" units="degE7">Latitude</field>
+      <field type="int32_t" name="lng" units="degE7">Longitude</field>
+      <field type="float" name="altitude" units="m">Altitude (MSL)</field>
+      <field type="float" name="roll" units="rad">Roll angle</field>
+      <field type="float" name="pitch" units="rad">Pitch angle</field>
+      <field type="float" name="yaw" units="rad">Yaw angle</field>
+      <field type="float" name="depth" units="m">Depth</field>
```

Comment:
rename "depth" to "distance".. and update comment.

Reply:
renamed "depth" to "distance" and changed description to, "Distance (uncorrected)"